### ISSUE TITLE
Breaking change: Error classes are exported as types

### ISSUE THREAD

**Issue by Sytten** [prisma/prisma#4365]
## Bug description

I used to be able to do:
```typescript
import { PrismaClientKnownRequestError } from '@prisma/client';

if (!(err instanceof PrismaClientKnownRequestError)) {
    ...
  }
``` 

But now the classes are exported as types so we can't use them in instanceof.

## Expected behavior

They should be exported as values.

## Environment & setup
- Prisma version: 2.12.0

**Comment by Sytten** [comment]
Problem lies here: https://github.com/prisma/prisma/blob/2.12.0/src/packages/client/src/generation/TSClient/common.ts#L77
Should have been `import` instead of `type`.

**Comment by DregondRahl** [comment]
`import { PrismaClientKnownRequestError } from '@prisma/client/runtime';` 

Bumped into the same issue, using that for now.

**Comment by timsuchanek** [comment]
This is released in `2.12.1`.
